### DISCUSSION TITLE
Failed to re-signin Copilot after revoking access in Authorized OAuth Apps

### DISCUSSION THREAD

**Discussion by davidlu1001** [community/community#15702] [Copilot Conversations]
Hi there,

I revoked the access for all the apps in `Authorized OAuth Apps` due to security issues (including `Copilot`).

Is it possible to re-add Copilot access to my Authorized OAuth Apps again?

I've tried reinstalling the VSCode extension, but nothing popup and still got:

![image](https://user-images.githubusercontent.com/1622825/165644724-93d0be73-63f1-4a1a-b60a-a2e0ad92c08e.png)

```
[INFO] [auth] [2022-04-27T22:40:42.434Z] Invalid copilot token: missing token: 401 
[ERROR] [default] [2022-04-27T22:40:42.436Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

Also tried to Signup again, but it said my account already enabled.

![image](https://user-images.githubusercontent.com/1622825/165644814-0735e03f-462e-457d-befc-c68c0cd3ee7b.png)

Cheers,
David

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by RexWzh** [reply]
> <img alt="Screen Shot 2021-11-30 at 8 03 57 am" width="475" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
> 
> Please try signing out, copilot will prompt you to sign back in.

Same problem. This doesn't work in my case (Ubuntu 20.04)

**Comment by davidlu1001** [comment]
Tried the following and the combinations:
- Github Sign Out / Sign In
- Uninstall and reinstall `Copilot` extension
- Install `Copilot Nightly` extension

Not sure which step took effect, but finally I got a pop-up prompt to re-sign In.
